frontend/src/main/components/EnrollmentData/EnrollmentHistoryGraph.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  Tooltip,
+  ResponsiveContainer,
+  Legend,
+} from "recharts";
+
+// Stryker disable all: This is not to be run until the tests are written as it is a placeholder.
```

Comment:
No, that's not how this works.   This isn't a placeholder; this is a component that you are planning to use later.  So it needs to be covered by tests.

If you put this in now, then later on, when you just "use the component", there's not way to know that we are using a component not covered by tests.  The exception will continue to be there.